test/common/common/hash_test.cc

```diff
@@ -27,6 +27,14 @@ TEST(Hash, murmurHash2_64) {
   EXPECT_EQ(6142509188972423790U, MurmurHash::murmurHash2_64(""));
 }
 
+TEST(Hash, crc16) {
+  EXPECT_EQ(44950, Crc16::crc16("foo"));
```

Comment:
for my info -- how did you calculate these CRC-16 values?

Reply:
I used the online CRC16 calculator: https://www.lammertbies.nl/comm/info/crc-calculation.html, then I cross check against a redis cluster using the CLUSTER KEYSLOT command to ensure we ended up with the same slot.

Reply:
excellent

Reply:
Please add a comment to this effect.